### PR TITLE
UserController component

### PR DESCRIPTION
**Pull request by maruxan**
- Created UserController component

### CODE REVIEW

File: src/components/UserController/UserController.module.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  font-weight: 400;
+}
+
+.ProfileText p::first-letter {
+  text-transform: uppercase;
+}
+
+.ProfileText h2 {
+  margin: 0;
+
```
Comment (monsterkosh) [inline src/components/UserController/UserController.module.css:38]: delete empty line

File: src/components/UserController/UserController.module.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    height: 100%;
+  }
+
+  .ProfileAvatar {
+    margin: 0 0 1.5rem 0;
+  }
+
+  .ProfileText h2 {
+    margin: 0 0 1rem 0;
+
```
Comment (monsterkosh) [inline src/components/UserController/UserController.module.css:96]: delete empty line

**Review by monsterkosh — CHANGES_REQUESTED** [review]
all good, just delete those 2 empty lines on the css

**Comment by maruxan** [comment]
Done

**Review by monsterkosh — APPROVED** [review]
All good !

**Review by JoaquinBruhnIntive — APPROVED** [review]
LGTM

**Pull request merged**